src/components/new-request/search.vue

```diff
@@ -389,13 +439,23 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     return (this.getRequestActionCd === NrRequestActionCodes.NEW_BUSINESS && this.request?.group === 1)
   }
 
+  get isPassAlterValidation (): boolean {
+    const legalType = this.business.legalType as string
```

Comment:
Hongjing, one other way that you can cast this since you want to compare to CorpTypeCd is:
`const legalType = this.business.legalType as unknown as CorpTypeCd`

This is instead of casting it as a string.